OneCommunityGlobal/HGNRest
# Sriram taking over for Imran added requestBio permission

# Description
Added Permission that gives the user permission to toggle the "Bio announcement" switch under "Reports" -> "Weekly Summaries Reports"


## Related PRS (if any):
To test this backend PR you need to checkout the [#2289](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2289) frontend PR.
…

## Main changes explained:
- Added new permission "requestBio" to toggle the bio announcement.

## How to test:
1. check into current branch
2. do `npm install` and `npm run build` to run this PR locally
3. Clear site data/cache
4. Follow steps on the Frontend PR [#2289](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2289)


## Note:
The User assigned the Permission should also have "See Weekly Summaries" permission to allow them to navigate to the Reports Page
Changing the Bio announcement for the "Edwardo Mentor" returns a 404 Error because "userProfile validation failed: warnings.3.description: uuttt is not a valid enum value for path description., warnings.4.description: uuttt is not a valid enum value for path description.". Please test for other Members.
